### PR TITLE
Update dev setup documentation

### PR DESCRIPTION
**Pull request by cavis**
Updating README with my experience installing the dev environment.

Also changes `database.yml` to allow a password on the test database ... which may or may not break the build.  We'll find out!

### CODE REVIEW

**Comment by cavis** [comment]
I thought I remembered blank values in database.yml for `username` or `password` causing problems.  But that was probably working with postgres, not mysql.  This seems to work fine.

File: README.md [hunk 1]
```diff
@@ -50,10 +53,13 @@ echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
 source ~/.bash_profile
 
 # ruby (.ruby-version default)
-rbenv install
+rbenv install 2.1.2
```
Comment (chrisrhoden) [inline README.md]: I think that if .ruby-version is present in the directory you run `rbenv install` in (which it is if you're in a checkout for this project) `rbenv install` is technically more correct than `rbenv install 2.1.2`. That said, the ordering on this is a little wonky, so maybe good to specify

File: README.md [hunk 2]
```diff
@@ -66,26 +72,44 @@ cd cms.prx.org
...
+
+After bootstrapping the prx.org schema into the test database, just run `bundle exec rake`.  Bingo!
+
+### Development server
+
+The dev server runs through powder - see a [full list of commands](https://github.com/Rodreegez/powder#working-with-pow) over there.
+
+```
+powder link
+open http://cms.prx.dev
```
Comment (farski) [inline README.md]: This should probably just be `powder open`

File: README.md [hunk 3]
```diff
@@ -66,26 +72,44 @@ cd cms.prx.org
...
+gunzip < 20150101_mediajoint_production_data.gz | mysql -u root cms_prx_org_development
+```
+
+### Testing
+
+After bootstrapping the prx.org schema into the test database, just run `bundle exec rake`.  Bingo!
+
+### Development server
+
+The dev server runs through powder - see a [full list of commands](https://github.com/Rodreegez/powder#working-with-pow) over there.
```
Comment (farski) [inline README.md]: "The dev server runs through pow", powder is just the gem

File: README.md [hunk 4]
```diff
@@ -31,18 +31,21 @@ These instructions are written assuming Mac OS X install.
 ```
 # Homebrew - http://brew.sh/
 ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/homebrew/go/install)"
+brew update
 
 # Git - http://git-scm.com/
 brew install git
```
Comment (farski) [inline README.md]: Might be nice to move the dependencies to a [Brewfile](https://robots.thoughtbot.com/brewfile-a-gemfile-but-for-homebrew)

**Pull request merged**